### DISCUSSION TITLE
problem with laravisists table

### DISCUSSION THREAD

**Discussion by mouawiashride** [coderflexx/laravisit#28] [General]
i add laravisit to model product but it doesn't track any thing
![b81981a2-b8e6-45f9-8c3a-8243411c1a2dfdges-1209509800](https://user-images.githubusercontent.com/85014842/212523095-979f7862-c28d-4ce1-ba26-5a957929d471.png)

**Comment by miftahrahmat** [comment]
I also experienced the same thing

**Reply by ElisameAraujo** [reply]
You need to pass the id of post, product, article, etc inside your controller were you are redirecting to view the post.

Example:

```
public function verPost($post){
        if(Posts::where('slugPost', $post)->exists()){

            $post = Posts::withCount([
                'comentarios' => function ($q){
                    $q->where('statusComentario', 2);
                }
            ])
            ->withTotalVisitCount() // method to count the total views of a page
            ->where('slugPost', $post)->first();
            $post->visit()->withIp()->dailyInterval(); // add this line inside the controller that shows the post

            ...
        }
    }
```

Adding  `->withTotalVisitCount()` you can access from your blade view the total of views, for example: `{{$post->visit_count_total}}`
